docs/general/foundation/testnet/README.md

```diff
@@ -36,6 +36,9 @@ Feel free to use other seed nodes besides the one provided here.
 
 * [Oasis Core](https://github.com/oasisprotocol/oasis-core) version:
   * [22.1.5](https://github.com/oasisprotocol/oasis-core/releases/tag/v22.1.5)
+  * To sync your node from the genesis, use the following Oasis
+    Core version first ([read more](upgrade-log.md#2022-04-04-upgrade)):
```

Comment:
The *Upgrade Log* page is quite dense. Would it make sense to create a general page or an entry under Node Operator FAQ for describing how Oasis Core handles syncing from Genesis?

Something along the lines of:

Use the first Oasis Core version to sync blocks from genesis onwards. After reaching the upgrade height _x_, the Oasis Node process will stop with a message similar to the following:
```
log message(s)
```
At that point you need to switch the Oasis Node binary with the one from the next Oasis Core version. After restarting the Oasis Node process, the syncing will resume.

Reply:
Revamped [handling-network-upgrades.md](https://github.com/oasisprotocol/docs/blob/matevz/feature/testnet-upgrade-info/docs/general/run-a-node/maintenance-guides/handling-network-upgrades.md) to include this.